_static/js/citesupport.es6

```diff
@@ -239,17 +242,15 @@ class CiteSupport {
      *
      * setCitations() wants this structure:
      *    [<citation_index>, <citation_string>, <citation_id>]
-     * 
+     *
      * @param {Object[]} rebuildData An array of values for insertion of citations into a document
      * @return {Object[]}
      */
     convertRebuildDataToCitationData(rebuildData) {
```

Comment:
This is flimsy because you aren't guaranteed to receive a return value of the same shape. I'd probably return an empty object here at the very least.